index.js

```diff
@@ -3,9 +3,15 @@
 const Client = require('./src/client').YotiClient;
 const AmlAddress = require('./src/aml_type').AmlAddress;
 const AmlProfile = require('./src/aml_type').AmlProfile;
+const Policy = require('./src/dynamic_policy_service').Policy;
```

Comment:
This variable and the corresponding module should become something like `AttributeList`